src/timed_event.cpp

```diff
@@ -51,8 +53,7 @@ void timed_event::actualize()
             if( rl_dist( u_pos, map_point ) <= 4 ) {
                 const mtype_id &robot_type = one_in( 2 ) ? mon_copbot : mon_riotbot;
 
-                g->u.add_memorial_log( pgettext( "memorial_male", "Became wanted by the police!" ),
-                                       pgettext( "memorial_female", "Became wanted by the police!" ) );
+                g->events().send( event::make<event_type::becomes_wanted>( g->u.getID() ) );
```

Comment:
Large parts of this are repeated over and over. Consider making a template function `send<X>` where `X` is the template  parameter used with `event::make`. So the usual invocation looks something like this:
```C++
g->events().send<event_type::becomes_wanted>( g->u.getID() );
```

Reply:
Indeed, good idea.  The disadvantage is that it's one more layer of template instantiation to step through when you pass the wrong argument type or similar, but it's probably worth it to reduce the repetition.